Documentation/source/changelog.rst

```diff
@@ -2,6 +2,13 @@
 Changelog
 =========
 
+3.1.0 (2017-08-25)
+==================
+
+- Adapted UI for iPad (landscape orientation is now supported on iPad)
+- Fixed issue on preview screen, when there is no orientation (device on flat position)
+- For iPhones we disabled automatic rotation of the picture on the preview view. Previously it was rotated when the device was held in landscape. Now the picture is always shown in portrait orientation on iPhones.
```

Comment:
IN the preview view probably. But ON might also be correct.

Reply:
Would also use "in" here.